tests/pytest.ini

```diff
@@ -1,13 +1,19 @@
 [pytest]
-log_level = DEBUG
+render_collapsed = True
 log_cli=true
 filterwarnings =
     ignore::urllib3.exceptions.InsecureRequestWarning
 markers =
```

Comment:
The marker `dependency` is missing to avoid the warning messages like: 
```
system/test_cv_configlet.py:261
  /projects/ansible-cvp/tests/system/test_cv_configlet.py:261: PytestUnknownMarkWarning: Unknown pytest.mark.dependency - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.dependency(depends=["authentication"], scope='class')
```

Reply:
I am not able to reproduce this output. Could you check you have latest requirements-dev.txt installed with the following packages ?

```bash
$ pip freeze | grep pytest
pytest==6.2.5
pytest-cov==3.0.0
pytest-dependency==0.5.1
pytest-html==3.1.1
pytest-metadata==1.11.0
```

Reply:
Indeed, I was missing `pytest-dependency==0.5.1`, my bad, all good.